user_test.go

```diff
@@ -19,8 +19,8 @@ func TestNaturalUserSave(test *testing.T) {
 }
 
 func createTestUser(serv *MangoPay) *NaturalUser {
-	user := serv.NewNaturalUser("Firstname", "Lastname", "example@example.com",
-		time.Date(2000, time.January, 01, 0, 0, 0, 0, time.UTC).Unix(), "EN", "EN")
+	user := serv.NewNaturalUser("Firstname", "Lastname", "example@example.com", "owner",
+		time.Date(2000, time.January, 0o1, 0, 0, 0, 0, time.UTC).Unix(), "EN", "EN", true)
```

Comment:
`0o1` what it means ?

Reply:
Now changed it to simply 1. Some how it was originally 01 and editor induced the change automatically , which is unexpected. when i simply make it 1 , it worked.

Reply:
done